dockers/docker-sonic-gnmi/gnmi-native.sh

```diff
@@ -17,6 +17,10 @@ X509=$(echo $TELEMETRY_VARS | jq -r '.x509')
 GNMI=$(echo $TELEMETRY_VARS | jq -r '.gnmi')
```

Comment:
Should we also make change in dockers/docker-sonic-telemetry/telemetry.sh?

Reply:
Yes, will create task for you